lib/tasks/maintenance.rake

```diff
@@ -19,6 +22,7 @@ namespace :maintenance do
   end
 
   task :update_bank_holidays do
-    `curl -k https://www.gov.uk/bank-holidays.json -o config/bank-holidays.json`
+    `curl -k https://www.gov.uk/bank-holidays.json -o \
+    config/bank-holidays.json`
```

Comment:
We could replace this entirely with [sh](http://rake.rubyforge.org/FileUtils.html#method-i-sh).